tests/wsl/firstrun.pm

```diff
@@ -61,7 +61,7 @@ sub license {
     if (is_sle) {
         # license warning
         assert_screen 'wsl-license-not-accepted';
-        send_key 'ret';
+        send_key 'ant-n';
```

Comment:
I guess a typo

Reply:
Nope, in fact for the SLED `wsl-license-not-accepted` the default selected option is `Yes` so the return key causes the tests to fail. This way, I believe it's more "universal" for all the tests to choose the `No`.

Reply:
Yes it is, in SLED it does appear a different screen for confirmation.